docs/configuration.md

```diff
@@ -72,7 +82,7 @@ Iceberg tables support table properties to configure table behavior, like the de
 | write.wap.enabled                  | false              | Enables write-audit-publish writes |
 | write.summary.partition-limit      | 0                  | Includes partition-level summary stats in snapshot summaries if the changed partition count is less than this limit |
 | write.metadata.delete-after-commit.enabled | false      | Controls whether to delete the oldest version metadata files after commit |
-| write.metadata.previous-versions-max       | 100        | The max number of previous version metadata files to keep before deleting after commit |
+| write.metadata.previous-versions-max      | 100        | The max number of previous version metadata files to keep before deleting after commit |
```

Comment:
Can you revert this whitespace-only change?

Reply:
Thank you for your reply. I have had revert the whitespace